src/main/java/org/folio/readingroom/service/UserService.java

```diff
@@ -6,4 +6,8 @@
 
 public interface UserService {
   void validatePatron(UUID patronId, List<PatronPermission> patronPermissions);
+
+  void validatePatronPermissions(UUID patronId, List<PatronPermission> patronPermissions);
```

Comment:
I don't see this method is being called anywhere outside of the class.
Either we can remove this from here and make this method private in the impl class or instead of calling validate patron , we can call below 2 methods seperately from updatePatronPermissionsByUserId method.
```
userService.validatePatronPermissions(patronId, patronPermissions);
userService.validatePatronExistence(patronId);
```
If we follow the latter approach, we can remove validatePatron method.

Reply:
fixed